nload/plan.sh

```diff
@@ -0,0 +1,41 @@
+pkg_name=nload
+pkg_origin=core
+pkg_version="0.7.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
```

Comment:
License is indicated as GPL-2.0 on the github page.  Could you please verify if it's 2.0+ or 2.0-only and update this to the corresponding identifier on https://spdx.org/licenses/